<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="5" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2018-05-24T14:51:22Z" name="Default suite" finished-at="2018-05-24T14:55:47Z" duration-ms="264922">
    <groups>
    </groups>
    <test started-at="2018-05-24T14:51:22Z" name="Default test" finished-at="2018-05-24T14:55:47Z" duration-ms="264922">
      <class name="io.branch.tests.TeamTests">
        <test-method is-config="true" signature="setup()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:51:22Z" name="setup" finished-at="2018-05-24T14:51:27Z" duration-ms="5272" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateBrokenLinksOnTeamPage()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:51:27Z" name="validateBrokenLinksOnTeamPage" description="Test Method to verify for presence of any broken links on Team page " finished-at="2018-05-24T14:51:51Z" duration-ms="23141" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[There are broken links detected on team page, details are mentioned in log expected [0] but found [25]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: There are broken links detected on team page, details are mentioned in log expected [0] but found [25]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at io.branch.tests.TeamTests.validateBrokenLinksOnTeamPage(TeamTests.java:224)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateBrokenLinksOnTeamPage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:51:51Z" name="tearDown" finished-at="2018-05-24T14:51:51Z" duration-ms="77" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:51:51Z" name="setup" finished-at="2018-05-24T14:51:55Z" duration-ms="4565" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateDeptCount()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:51:55Z" name="validateDeptCount" description="Test Method to verify if department count matches" finished-at="2018-05-24T14:52:00Z" duration-ms="5218" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDeptCount -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:52:00Z" name="tearDown" finished-at="2018-05-24T14:52:03Z" duration-ms="2310" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:52:03Z" name="setup" finished-at="2018-05-24T14:52:07Z" duration-ms="4373" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateEmployeeCount()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:52:07Z" name="validateEmployeeCount" description="Test Method to verify the count of Employees match between All tab and rest of the departments Tab" finished-at="2018-05-24T14:52:50Z" duration-ms="42584" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failing due to employee count mismatch between All Tab and rest of all departments expected [143] but found [138]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failing due to employee count mismatch between All Tab and rest of all departments expected [143] but found [138]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at io.branch.tests.TeamTests.validateEmployeeCount(TeamTests.java:105)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmployeeCount -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:52:50Z" name="tearDown" finished-at="2018-05-24T14:52:52Z" duration-ms="2313" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:52:52Z" name="setup" finished-at="2018-05-24T14:52:57Z" duration-ms="4573" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateEmployeeImages()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:52:57Z" name="validateEmployeeImages" description="Test Method to if all employee has images" finished-at="2018-05-24T14:53:37Z" duration-ms="40690" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmployeeImages -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:53:37Z" name="tearDown" finished-at="2018-05-24T14:53:40Z" duration-ms="2313" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:53:40Z" name="setup" finished-at="2018-05-24T14:53:44Z" duration-ms="4546" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateEmployeeNames()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:53:44Z" name="validateEmployeeNames" description="Test Method to verify the employee names across all tabs and all departments" finished-at="2018-05-24T14:54:27Z" duration-ms="43181" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmployeeNames -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:54:27Z" name="tearDown" finished-at="2018-05-24T14:54:30Z" duration-ms="2308" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:54:30Z" name="setup" finished-at="2018-05-24T14:54:34Z" duration-ms="4487" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateMissingEmployeeNamesInDeptTabs()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:54:34Z" name="validateMissingEmployeeNamesInDeptTabs" description="Test method to verify missing employees in individual departments but present in all tab" finished-at="2018-05-24T14:55:34Z" duration-ms="60412" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateMissingEmployeeNamesInDeptTabs -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:55:34Z" name="tearDown" finished-at="2018-05-24T14:55:37Z" duration-ms="2313" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:55:37Z" name="setup" finished-at="2018-05-24T14:55:41Z" duration-ms="4686" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateSignUpForDemoRequest()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:55:41Z" name="validateSignUpForDemoRequest" description="Test method to complete sign up process for branch demo request" finished-at="2018-05-24T14:55:45Z" duration-ms="3449" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateSignUpForDemoRequest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:io.branch.tests.TeamTests@291b4bf5]" started-at="2018-05-24T14:55:45Z" name="tearDown" finished-at="2018-05-24T14:55:47Z" duration-ms="2072" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- io.branch.tests.TeamTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
